packages/desktop-client/src/components/modals/CreateLocalAccount.tsx

```diff
@@ -26,7 +33,7 @@ function CreateLocalAccount({ modalProps, actions }) {
   let validateBalance = balance => !isNaN(parseFloat(balance));
 
   return (
-    <Modal title="Create Local Account" {...modalProps} showBack={false}>
```

Comment:
"showBack" is not a valid property for Modal - it does nothing.

The Back button logic is in this component.